src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.props

```
@@ -198,4 +198,15 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="IncludeSymbolFilesInNativeAOTPackage"
+          Condition="'$(BuildNativeAOTRuntimePack)' == 'true'"
+          AfterTargets="GetFilesToPackage">
+    <ItemGroup>
+      <!-- ILC needs the symbols next to the binaries -->
+      <FilesToPackage Update="@(FilesToPackage)" Condition="'%(FilesToPackage.IsSymbolFile)' == 'true'">
```

Using an item function has better perf and is more expressive:

```suggestion
      <FilesToPackage Update="@(FilesToPackage->WithMetadataValue('IsSymbolFile', 'true'))">
```

---

Thanks.